tests/test_generate_diff.py

```diff
@@ -3,7 +3,7 @@
 from tests.paths import resolve_path
 
 
-def test_generate_diff() -> None:
+def test_parse_json() -> None:
```

Comment:
it's the same as `test_generate_diff_json`, consider removing (and it's expected diff too)